docs/src/content/docs/challenges/nx/27-forbid-enum-rule.md

```diff
@@ -20,7 +20,7 @@ You will also need to write tests to verify the rule's functionality.
 
 The `tools/eslint-rules` folder contains the starter code for this challenge.
 
-To test the rule inside your project, add `"@nrwl/nx/workspace/forbidden-enum": "error"` to the `eslintrc.json`. You can navigate to Challenge 47, `Enums vs. Union Types', and you should immediately see an error.
+To test the rule inside your project, add `"@nx/workspace/forbidden-enum": "error"` to the `eslintrc.json`. You can navigate to Challenge 47, `Enums vs. Union Types', and you should immediately see an error.
```

Comment:
can you create a PR for that please 🙏

Reply:
@jdegand fixed it yesterday: https://github.com/tomalaforge/angular-challenges/commit/008051cab8264fa813b90d4778a66201e3cccde0